src/mame/heathkit/tlb.h

```diff
@@ -39,11 +39,28 @@ class heath_tlb_device : public device_t
 	virtual void device_add_mconfig(machine_config &config) override;
 	virtual void device_resolve_objects() override;
 
-private:
-
 	void mem_map(address_map &map);
 	void io_map(address_map &map);
 
+	required_device<cpu_device> m_maincpu;
+
+	// Clocks
+	static constexpr XTAL MASTER_CLOCK = XTAL(12'288'000);
+
+	// Standard H19 used a 2.048 MHz clock
+	static constexpr XTAL H19_CLOCK = MASTER_CLOCK / 6;
+	static constexpr XTAL MC6845_CLOCK = MASTER_CLOCK / 8;
+	static constexpr XTAL INS8250_CLOCK = MASTER_CLOCK / 4;
+
+	// Beep Frequency is 1 KHz
+	static constexpr XTAL H19_BEEP_FRQ = (H19_CLOCK / 2048);
+
+	// Capacitor value in pF
+	static constexpr uint32_t H19_KEY_DEBOUNCE_CAPACITOR = 5000;
```

Comment:
Do these need to be in the header?  If they’re not actually used by anything in the class declaration they shouldn’t need to be here.  Also, one of them is shadowed by a similarly named constant in the .cpp file.